crates/ruff/src/rules/pyflakes/snapshots/ruff__rules__pyflakes__tests__del_shadowed_import_shadow_in_local_scope.snap

```diff
@@ -17,13 +17,12 @@ source: crates/ruff/src/rules/pyflakes/mod.rs
 4 3 | def f():
 5 4 |     os = 1
 
-<filename>:12:9: F811 Redefinition of unused `os` from line 2
-   |
-10 |     def g():
-11 |         # `import os` should still be flagged as shadowing an import.
-12 |         os = 1
-   |         ^^ F811
-13 |         print(os)
```

Comment:
I think it's fine not to flag this.

Reply:
Why did this change? Is it because `f` assigned `os` on line 5?

Should we update the comment in the test input file?

Reply:
I did!

Reply:
It's because we `del os` in the containing scope (not visible in this diff).